### PR TITLE
fix(transaction-tokens): show none AE related transactions correctly

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

**Review by konradodo — APPROVED** [review]

**Review by peronczyk — APPROVED** [review]

File: src/popup/components/TransferReview.vue [hunk 1]
```diff
@@ -129,6 +129,10 @@ export default {
       'tippingV1',
       'tippingV2',
     ]),
+    isAex9() {
```
Comment (peronczyk) [inline src/popup/components/TransferReview.vue:132]: I suggest to use more informative naming here. Like for example `isSelectedAssetAex9`.

File: src/popup/components/TransferReview.vue [hunk 2]
```diff
@@ -269,6 +273,7 @@ export default {
             contractId: this.tippingContract.deployInfo.address,
             type: SCHEMA.TX_TYPE.contractCall,
             function: 'tip',
+            selectedTokenId: selectedAsset.contractId,
```
Comment (peronczyk) [inline src/popup/components/TransferReview.vue:276]: I see some possible naming collision here. One one side we have `tokenId` which suggests that this is the `id` of the token. On the right side it is a `contractId` which is a different thing.
The `IToken` type has the `id` and `contractId` and I guess those 2 things should not be mixed.

Comment (CedrikNikita) [reply src/popup/components/TransferReview.vue:276]: There is no such thing as `tokenId` in the whole wallet. In this case we have an id of a contract that was used to sent tip and id of token (which is also a contract) that was sent.

Comment (peronczyk) [reply src/popup/components/TransferReview.vue:276]: Discussed in private.

Comment (CedrikNikita) [reply src/popup/components/TransferReview.vue]: 346de93245cff36e58f5df570362a1f2e136866c

**Review by peronczyk — APPROVED** [review]

**Pull request merged**